consensus/ibft/extra.go

```diff
@@ -77,7 +77,7 @@ func getIbftExtra(h *types.Header) (*IstanbulExtra, error) {
 // IstanbulExtra defines the structure of the extra field for Istanbul
 type IstanbulExtra struct {
 	Validators    []types.Address
-	Seal          []byte
+	ProposerSeal  []byte
```

Comment:
Why rename? It comes from document?

Reply:
Yes, @dbrajovic and I decided it's much clearer in code (and for us all) that we have a clear distinction between _proposer seals_ and _commit seals_